bitnami/nginx/templates/ingress.yaml

```diff
@@ -0,0 +1,38 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "fullname" . -}}
+{{- $ingressPath := .Values.ingress.path -}}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{ $fullName }}
+  labels:
+    app: {{ $fullName }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+{{- with .Values.ingress.annotations }}
```

Comment:
On other charts we included the parameter `ingress.certManager` that allows setting the corresponding annotations for people using cert-manager to handle the certificates (see https://github.com/bitnami/charts/blob/master/upstreamed/wordpress/values.yaml#L214):

```yaml
    {{- if .Values.ingress.certManager }}
    kubernetes.io/tls-acme: "true"
    {{- end }}
```

Here you have an example: https://github.com/bitnami/charts/blob/master/upstreamed/wordpress/templates/ingress.yaml#L11

Reply:
done